File: Programing_Basics_with_JS/conditionalStatementsAdvancedMoreExercises/07.SchoolCamp.js
function schoolCamp(input) {
    let season = input[0];
    let typeOfGroup = input[1];
    let numbersOfStudents = Number(input[2]);
    let numbersOfNight = Number(input[3]);

    let pricePerNight = 0;
    let sport = '';

    switch(season) {
        case 'Winter':
            switch(typeOfGroup){
                case 'boys':
                    pricePerNight = 9.60;
                    sport = 'Judo';
                    break;
                case 'girls':
                    pricePerNight = 9.60;
                    sport = 'Gymnastics';
                    break;
                case 'mixed':
                    pricePerNight = 10;
                    sport = 'Ski';
                    break;
            } break;
        case 'Spring':
            switch(typeOfGroup){
                case 'boys':
                    pricePerNight = 7.20;
                    sport = 'Tennis';
                    break;
                case 'girls':
                    pricePerNight = 7.20;
                    sport = 'Athletics';
                    break;
                case 'mixed':
                    pricePerNight = 9.50;
                    sport = 'Cycling';
                    break;
            } break;
        case 'Summer':
            switch(typeOfGroup){
                case 'boys':
                    pricePerNight = 15;
                    sport = 'Football';
                    break;
                case 'girls':
                    pricePerNight = 15;
                    sport = 'Volleyball';
                    break;
                case 'mixed':
                    pricePerNight = 20;
                    sport = 'Swimming';
                    break;
            } break;
    }

    let allNeededMoney = pricePerNight * numbersOfNight * numbersOfStudents;

    if (numbersOfStudents >= 10 && numbersOfStudents < 20) {
        allNeededMoney = allNeededMoney * 0.95;
    } else if (numbersOfStudents >= 20 && numbersOfStudents < 50) {
        allNeededMoney = allNeededMoney * 0.85;
    } else if (numbersOfStudents >= 50) {
        allNeededMoney = allNeededMoney * 0.5;
    }

    console.log(`${sport} ${allNeededMoney.toFixed(2)} lv.`);
}

schoolCamp(['Spring', 'girls', '20', '7'])
schoolCamp(['Winter', 'mixed', '9', '15'])
schoolCamp(['Summer', 'boys', '60', '7'])
schoolCamp(['Spring', 'mixed', '17', '14'])

File: Programming_Basics_with_Python/conditional_statements_advanced_more_exercises/07._school_camp.py
season = input()
group_type = input()
numbers_of_students = int(input())
number_of_nights = int(input())

price_per_night = 0
sport_type = ''

if season == 'Winter':
    if group_type == 'boys' or group_type == 'girls':
        price_per_night = 9.60
        if group_type == 'boys':
            sport_type = 'Judo'
        elif group_type == 'girls':
            sport_type = 'Gymnastics'
    elif group_type == 'mixed':
        price_per_night = 10
        sport_type = 'Ski'
elif season == 'Spring':
    if group_type == 'boys' or group_type == 'girls':
        price_per_night = 7.20
        if group_type == 'boys':
            sport_type = 'Tennis'
        elif group_type == 'girls':
            sport_type = 'Athletics'
    elif group_type == 'mixed':
        price_per_night = 9.50
        sport_type = 'Cycling'
elif season == 'Summer':
    if group_type == 'boys' or group_type == 'girls':
        price_per_night = 15
        if group_type == 'boys':
            sport_type = 'Football'
        elif group_type == 'girls':
            sport_type = 'Volleyball'
    elif group_type == 'mixed':
        price_per_night = 20
        sport_type = 'Swimming'

hotel_fee = numbers_of_students * number_of_nights * price_per_night

if numbers_of_students >= 50:
    hotel_fee = hotel_fee * 0.5
elif 50 > numbers_of_students >= 20:
    hotel_fee = hotel_fee * 0.85
elif 20 > numbers_of_students >= 10:
    hotel_fee = hotel_fee * 0.95

print(f'{sport_type} {hotel_fee:.2f} lv.')